src/constants.ts

```diff
@@ -93,10 +95,31 @@ const PACT_CELO_MAINNET: FarmBotConfig = {
   ],
 }
 
+const UBE_CELO_MAINNET: FarmBotConfig = {
+  name: 'UBE-CELO-ubeswap',
+  abi: UBESWAP_SINGLE_REWARD_FARM_BOT_ABI,
```

Comment:
The compounder will work for any farm that inherits from `RevoUniswapStakingTokenStrategy` at the moment, so this works, but we'll definitely need to rethink things once we support farms with different `compound` signatures.